asset/api/Asset.Samples.Tests/Asset.Samples.Tests.csproj

```
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>

```

OK, so netstandard* is a specification, and then you have concrete implementations of that which target different platforms etc.
Libraries should almost always target netstandard* so they can be used by all implementations, as where "running" projects, like test projects should target implementations. In samples, we are shifting to targetting netcoreapp3.1.
```suggestion
    <TargetFramework>netcoreapp3.1</TargetFramework>
```